pkgs/development/cuda-modules/setup-hooks/extension.nix

```diff
@@ -53,7 +53,7 @@ final: _: {
   autoAddCudaCompatRunpathHook =
     final.callPackage
       (
-        {makeSetupHook, cuda_compat}:
+        {makeSetupHook, cuda_compat ? throw "autoAddCudaCompatRunpathHook: No cuda_compat for CUDA ${final.cudaMajorMinorVersion}" }:
```

Comment:
So to make sure I understand correctly, it's ok to use `throw` here because we do want to prevent Ofborg from evaluating the hook on platforms that don't support `cuda_compat`? Or should this be an abort as well?

Reply:
Indeed that's right :)